repos/spack_repo/builtin/packages/superlu/package.py

```diff
@@ -139,77 +135,4 @@ def _make_hdr_for_test(self, lib):
             "CFLAGS     = -O3 -DNDEBUG -DUSE_VENDOR_BLAS -DPRNTlevel=0 -DAdd_",
             "NOOPTS     = -O0",
         ]
-
-
-class CMakeBuilder(AnyBuilder, cmake.CMakeBuilder):
-    def cmake_args(self):
-        if self.pkg.version > Version("5.2.1"):
-            _blaslib_key = "enable_internal_blaslib"
-        else:
-            _blaslib_key = "enable_blaslib"
-        args = [
-            self.define(_blaslib_key, False),
-            self.define("CMAKE_INSTALL_LIBDIR", self.prefix.lib),
-            self.define_from_variant("CMAKE_POSITION_INDEPENDENT_CODE", "pic"),
-            self.define("enable_tests", self.pkg.run_tests),
-            self.define_from_variant("enable_fortran", "fortran"),
-        ]
-
-        if self.spec.satisfies("@6:"):
-            args.extend(
-                [
-                    self.define("TPL_ENABLE_METISLIB", True),
-                    self.define("TPL_METIS_INCLUDE_DIRS", self.spec["metis"].prefix.include),
-                    self.define("TPL_METIS_LIBRARIES", self.spec["metis"].libs),
-                ]
-            )
         return args
```

Comment:
This return statement should be in `cmake_args`, not in `setup_standalone_tests`